lib/vaultex/redirectable_requests.ex

```diff
@@ -1,56 +1,43 @@
 defmodule Vaultex.RedirectableRequests do
```

Comment:
We don't redirect here anymore at all, should this module be renamed?

Reply:
I kept it mainly for legacy reasons; but Req by default *does* follow redirects, so technically the name is still accurate (if a bit wordy).